guide/secure_secret_management.md

```diff
@@ -87,11 +88,11 @@ functions:
     image: functions/api-key-protected:latest
 ```
 
-Note that unlike the `envVars` in the first example, we do not provide the secret value, just a list of names: `"secrets": ["secret_api_key"]`. The secret value has already been securely stored in the Docker swarm.  One really great result of this type of configuration is that you can simplify your function code by always referencing the same secret name, no matter the environment, the only change is how the environments are configured.
+Note that unlike the `envVars` in the first example, we do not provide the secret value, just a list of names: `"secrets": ["secret_api_key"]`. The secret value has already been securely stored in the Docker swarm. One really great result of this type of configuration is that you can simplify your function code by always referencing the same secret name, no matter the environment, the only change is how the environments are configured.
 
-## Advanced Swarm Secrets
+<!-- ## Advanced Swarm Secrets
```

Comment:
Is this commented out?

Reply:
yes, this isn't supported anymore.  Before we merged the updated behavior, i had brought this up, but the new behavior was merged first